Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build
+FROM golang:latest AS base
+
+ENV HOME=/k8ify
+
+WORKDIR ${HOME}
+
+FROM base AS build
+
+COPY . ${HOME}
+
+RUN make test && go build -v .
```

Comment:
Ideally the tests would have run previously in the pipeline and are just a waste of time at this point.

Would also allow us to
- `BUILDKIT_CONTEXT_KEEP_GIT_DIR`
- add the entire `tests/` directory to `.dockerignore` (so they're not even sent to the docker daemon as the build context)